package.json

```diff
@@ -5,51 +5,42 @@
   "scripts": {
     "help": "webpack --help",
     "generate": "src/generate.js example.yml",
-    "build": "webpack --env prod",
-    "build-dev": "webpack --env dev",
+    "build": "lerna run build",
+    "build-dev": "lerna run build-dev",
     "clean": "rm -f lib/*",
-    "example": "webpack-dev-server --content-base example/ --host localhost --port 5005 --colors --watch",
+    "example": "webpack-dev-server --content-base ./example --host localhost --port 5005 --env dev --colors --watch --config webpack.example.config.js",
     "blanket": "webpack-dev-server --host localhost --port 6001 --colors --watch",
-    "test": "webpack --env prod && grunt karma:prod"
-  },
-  "dependencies": {
-    "native-promise-only": "^0.8.1",
```

Comment:
This is still being used in client.js

Reply:
@wizehive-seanburke what are you running to build it? the command should be `lerna run build`. I will work on documentation tomorrow, to address this, and what @evert0n pointed out above. Thanks for the feedback, y'all.

Reply:
The error was from running that command in the base folder.

I was changing up node versions with nvm so I removed the node_modules folder, did a `npm install` then ran `npm build`, which runs `lerna run build`. During the build it started complaining about that dep missing.

Reply:
you need to run lerna bootstrap, I believe.

Reply:
That was it, I assumed it would automatically check and do anything like that on a build run due to the lerna.json file.

Reply:
There's a little _lerna-ing_ curve, eh? *slaps knee...